src/Stryker.Core/Stryker.Core/Options/StrykerOptions.cs

```diff
@@ -88,5 +94,20 @@ private ThresholdOptions ValidateThresholds(int thresholdHigh, int thresholdLow,
 
             return new ThresholdOptions(thresholdHigh, thresholdLow, thresholdBreak);
         }
+
+        private List<string> ValidateFilesToExclude(string filesToExclude)
+        {
+            var excludedFiles = new List<string>();
+            try
+            {
+                excludedFiles = JsonConvert.DeserializeObject<List<string>>(filesToExclude);
```

Comment:
It would be nice if we could warn users on non-existant files but I don't think we should mark it as an invalid configuration. Logging as warning should be fine.